internal/security/secretstore/init.go

```diff
@@ -440,6 +464,61 @@ func (b *Bootstrap) BootstrapHandler(ctx context.Context, _ *sync.WaitGroup, _ s
 
 }
 
+func getKnownSecretsToAdd() (map[string][]string, error) {
+	// Process the env var for adding known secrets to the specified services' secret stores.
+	// Format of the env var value is:
+	//   "<secretName>[<serviceName>;<serviceName>; ...], <secretName>[<serviceName>;<serviceName>; ...], ..."
+	knownSecretsToAdd := map[string][]string{}
+
+	addKnownSecretsValue := strings.TrimSpace(os.Getenv(addKnownSecretsEnv))
+	if len(addKnownSecretsValue) == 0 {
+		return knownSecretsToAdd, nil
+	}
+
+	serviceNameRegx := regexp.MustCompile(ServiceNameValidationRegx)
+	knownSecrets := strings.Split(addKnownSecretsValue, ",")
+	for _, secretSpec := range knownSecrets {
+		// each secretSpec has format of "<secretName>[<serviceName>;<serviceName>; ...]"
+		secretItems := strings.Split(secretSpec, "[")
+		if len(secretItems) != 2 {
+			return nil, fmt.Errorf("invalid specification for %s environment vaiable: Format of value '%s' is invalid. Missing or too many '['", addKnownSecretsEnv, secretSpec)
```

Comment:
replace `[` with the named constant

Reply:
Not, for single occurrence. Reads just fine.